### PR TITLE
changes in comments,profile

### PR DESCRIPTION
**Pull request by preethasri**
made major changes in adding, editing the comment in single post and post card

made changes in backend bookmark controller
made changes in post controller
made changes in updating the profile

### CODE REVIEW

**Review by msdeshmukh009 — APPROVED** [review]
Suggestions:
- In the signup form, the input text is not visible.
- For sorting posts, you need to add the treading option as well.
- Edit comment is creating a new comment and not editing an existing comment.
- Change test credentials to something else.

File: src/features/user/userSlice.js [hunk 1]
```diff
@@ -62,13 +62,15 @@ export const removeBookmark = createAsyncThunk(
   async (arg, { rejectWithValue }) => {
     try {
       const { data, status } = await removeBookmarkService(arg);
-
+      console.log("hii")
```
Comment (mimansha-swarup) [inline src/features/user/userSlice.js:65]: remove unrequired consoles

File: src/features/user/userSlice.js [hunk 2]
```diff
@@ -62,13 +62,15 @@ export const removeBookmark = createAsyncThunk(
...
       if (status === 200) {
         toast.success("Post removed from bookmark");
         return data.bookmarks;
       }
-    } catch {
-      return rejectWithValue([], "Error occured. Try again later.");
+      
+    } 
+    catch(err) {
+      console.log(err)
```
Comment (mimansha-swarup) [inline src/features/user/userSlice.js:73]: suggestion: you can use console.error

**Review by mimansha-swarup — COMMENTED** [review]
few suggestion:
in suggestion bar and profile page avatar is coming in oval shape
single post page  contents are overflowing to right
and comments are not much visible try giving  darker color

image

File: src/features/user/userSlice.js (hunk 2 above)
Comment (preethasri) [reply src/features/user/userSlice.js:73]: sure will work on this

**Comment by preethasri** [comment]
> Suggestions:
> 
> * In the signup form, the input text is not visible.
> * For sorting posts, you need to add the treading option as well.
> * Edit comment is creating a new comment and not editing an existing comment.
> * Change test credentials to something else.

will make changes in this

**Pull request merged**